client/oci_env/commands.py

```diff
@@ -164,3 +172,9 @@ def poll(args, client):
 
 def pulp(args, client):
     client.exec(["pulp"] + args.command, interactive=True)
+
+
+
+def edit_env(args):
+    editor = os.getenv("EDITOR", "vim")
```

Comment:
Should we default it to a "more" common editor, like `vi`?
Not all distributions come with vim installed by default.